distributed/nanny.py

```diff
@@ -97,7 +99,6 @@ def __init__(
         services=None,
         name=None,
         memory_limit="auto",
-        memory_terminate_fraction: float | Literal[False] | None = None,
```

Comment:
This is a very recent addition so I think it's safe not to have a deprecation cycle